src/views/IndividualTeamView.vue

```diff
@@ -4,6 +4,7 @@
       <h3>{{ errorMessage }}</h3>
     </div>
     <div v-if="!error && loaded">
+      <page-title :title="'Test'" :path="{ name: 'CurrentReservations'}" />
```

Comment:
What should this actually say? Not "Test" I imagine.

Reply:
Good catch, thanks. That's supposed to be the team name. Fixing now